File: RedSheet.DbEntities/Models/vTeamMember.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vTeamMembers", Schema = "dbo")]
	[Serializable]
	public partial class vTeamMember  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int UserId { get; set; } 
 
	[Required] 
	[MaxLength(201)]
	public string UserName { get; set; } 

	public vTeamMember()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-team-member.ts
import { required, maxLength, range } from '@rx/annotations';

export class vTeamMember {
    constructor(vTeamMember?: vTeamMember )  {
        let properties = [ "userId", "userName",];
        for (let property of properties)
            if (vTeamMember && vTeamMember[property])
                this[property] = vTeamMember[property];
    }
 
	userId : number =   0 ;
 
    @required()
    @maxLength(201)
	userName : string =   undefined;
}
